Editor/AGS.Editor/DataFileWriter.cs

```diff
@@ -1282,6 +1282,13 @@ public static bool SaveThisGameToFile(string fileName, Game game, CompileMessage
             writer.Write(NativeConstants.GAME_DATA_VERSION_CURRENT);
             writer.Write(AGS.Types.Version.AGS_EDITOR_VERSION.Length);
             WriteString(AGS.Types.Version.AGS_EDITOR_VERSION, AGS.Types.Version.AGS_EDITOR_VERSION.Length, writer);
+            // Write extended engine caps; none for this version
```

Comment:
If I understand your idea right, this is to make release builds refuse to run games made with non-release builds?
Considering we often have breaking changes in the process of development, this may have certain sense.
What are the negative consequences:
* we must enforce the rule that we remove this flag as soon as we make first public release.
* if someone makes a game using nightly build from the build server, that game won't be runnable by final release even if technically they are compatible.